### ISSUE TITLE
onUploadProgress won't display correct loaded prop

### ISSUE THREAD

**Issue by otaviobps** [axios/axios#2706]
**Describe the bug**

When displaying a progress bar for a file upload, using the `onUploadProgress`, the `loaded` prop is always the total size of the file.

**To Reproduce**

Trying to implement the upload like this:

```
api.post("/private/customer/user/kyc", data, {
            onUploadProgress: function(progressEvent) {
                const progress = parseInt(
                    Math.round(
                        (progressEvent.loaded / progressEvent.total) * 100
                    )
                );
                setFileResponse({ ...fileResponse, progress });
            }
        })
            .then(response => {
                setFileResponse({
                    ...fileResponse,
                    uploaded: true,
                    url: response.data.identity.url
                });
                setDocumentSent(true);
            })
            .catch(() => setFileResponse({ ...fileResponse, error: true }));
    };
```

However `progressEvent.loaded` is always the total size of the file, resulting in always 100% loaded, even when the upload starts.

**Expected behavior**

Expected behavior would be the `progressEvent.loaded` to increase while the file is being uploaded.

**Environment:**
 - Axios Version [e.g. 0.18.0] `0.19.1`
 - OS: [e.g. iOS 12.1.0, OSX 10.13.4] : `OSX 10.15.2`
 - Browser [e.g. Chrome, Safari] `Chrome, Safari or Firefox`
 - Browser Version [e.g. 22] `All latest versions`
 - Additional Library Versions [e.g. React 16.7, React Native 0.58.0] `React 16.12.0`

**Comment by alec-joy** [comment]
Are you uploading files to box.com by any chance?

**Comment by otaviobps** [comment]
@alec-joy, no, I'm uploading to my backend, then to aws

**Comment by chinesedfan** [comment]
Closed in favor of #2709.
